fe/fe-core/src/main/java/com/starrocks/load/loadv2/BulkLoadJob.java

```diff
@@ -237,9 +238,19 @@ protected List<TabletFailInfo> getTabletFailInfos() {
         return failInfos;
     }
 
+    /**
+     * Not retryable cases
+     * 1. already retry too many times
+     * 2. BE parses data error, such as invalid json
+     */
+    protected boolean isRetryable(String failMsg) {
```

Comment:
should not retry if it's cancelled by user where the failMsg is something like "user cancel" ?

Reply:
https://github.com/StarRocks/starrocks/blob/main/fe/fe-core/src/main/java/com/starrocks/load/loadv2/LoadJob.java#L724
`user cancel` will not retry job actually.

Reply:
I add the type check for safety.